github-webhook-handler.d.ts

```diff
@@ -3,16 +3,16 @@
 import { IncomingMessage, ServerResponse } from "http";
 import { EventEmitter } from "events";
 
-interface CreateHandlerOptions {
+export interface CreateHandlerOptions {
     path: string;
     secret: string;
     events?: string | string[];
 }
 
-interface handler extends EventEmitter {
+export interface handler extends EventEmitter {
     (req: IncomingMessage, res: ServerResponse, callback: (err: Error) => void): void;
 }
 
 declare function createHandler(options: CreateHandlerOptions|CreateHandlerOptions[]): handler;
 
-export = createHandler;
```

Comment:
is this a necessary change?
I'm not expert enough to know the difference between these two, maybe there isn't one?

Reply:
That might've been a miscommit. I'm using the octokit we hook now anyway, so no longer an issue for me.